modules/gapi/perf/cpu/gapi_imgproc_perf_tests_fluid.cpp

```diff
@@ -125,6 +125,24 @@ INSTANTIATE_TEST_CASE_P(SobelPerfTestFluid32F, SobelPerfTest,
             Values(1, 2),
             Values(cv::compile_args(IMGPROC_FLUID))));
 
+INSTANTIATE_TEST_CASE_P(SobelXYPerfTestFluid, SobelXYPerfTest,
+    Combine(Values(AbsExact().to_compare_f()),
+            Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1),
+            Values(3),                                     // add 5x5 once supported
```

Comment:
`FIXME` or `TODO` is a good tag for `grep`-pable comments. Otherwise, we'll never find this again (only occasionally)

Reply:
Added TODO mark in comments